src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -95,10 +97,9 @@ export const TradeConfirm = () => {
   const bestSwapper = bestSwapperWithQuoteMetadata?.swapper
 
   const reset = useCallback(() => {
-    setValue('buyTradeAsset.amountCryptoPrecision', '')
-    setValue('sellTradeAsset.amountCryptoPrecision', '')
-    setValue('fiatSellAmount', '')
-  }, [setValue])
+    swapperDispatch({ type: SwapperActionType.CLEAR_AMOUNTS })
+    swapperDispatch({ type: SwapperActionType.SET_VALUES, payload: { fiatSellAmount: '' } })
```

Comment:
Do we want a generalized `RESET` action? Instead of clearing in one and setting to a base value in the other?